tests/end_to_end/tests.py

```diff
@@ -249,6 +250,21 @@ def step_autoconf(self):
                 auto_controller.app = app
                 auto_controller._default()
 
+    def step_add_service_endpoints(self, client_id):
+        endpoint_controller = EndpointController()
+        for security_server in self.config["security_server"]:
+            configuration = endpoint_controller.create_api_config(security_server, self.config)
+            for client in security_server["clients"]:
+                for service_description in client["service_descriptions"]:
+                    for service in service_description["services"]:
+                        for endpoint in service[0]["endpoints"]:
+                            endpoint_controller.remote_add_service_endpoints(configuration, security_server, client, service_description, endpoint)
+        description = get_service_description(self.config, client_id)
+        assert len(description["services"][0]["endpoints"]) == 1
```

Comment:
This should be "assert len(description["services"][0]["endpoints"]) == 5" as there are 4 generated endpoints in addition to the one you are adding as this description is a real OpenAPI3 description with some endpoints to be generated